resources/web/docs.js

```diff
@@ -637,6 +638,7 @@
 
   function init_headers() {
     // Add on-this-page block
+    var this_page = jQuery('<div id="this_page"></div>').appendTo(right_col);
```

Comment:
Do we want to append this element to `right_col`, or insert it somewhere near the top or in the middle of the `right_col` element's children? I ask because in the linked issue, it looks like the "On the page" block appears above the table of contents:

![image](https://user-images.githubusercontent.com/1238659/57258095-35729480-7010-11e9-9e5b-5b379ea29344.png)

And when I tested locally, it appears below the table contents, which pretty much buries it in terms of user experience:

![image](https://user-images.githubusercontent.com/1238659/57258112-41f6ed00-7010-11e9-9890-e48bb3057539.png)